subsys/net/l2/ethernet/eth_stats.h

```diff
@@ -70,9 +89,13 @@ static inline void eth_stats_update_pkts_tx(struct net_if *iface)
 static inline void eth_stats_update_broadcast_rx(struct net_if *iface)
 {
 	struct net_stats_eth *stats;
+	const struct ethernet_api *api = ((const struct ethernet_api *)
+		net_if_get_device(iface)->driver_api);
 
-	stats = ((const struct ethernet_api *)
-		 net_if_get_device(iface)->driver_api)->stats;
+	if (!api->get_stats)
```

Comment:
Please make sure that all the one line "if" statements have {}

Reply:
Sorry, missed this one.